src/plugins/TokenReceiverPlugin.sol

```diff
@@ -59,12 +59,21 @@ contract TokenReceiverPlugin is BasePlugin, IERC721Receiver, IERC1155Receiver {
         PluginManifest memory manifest;
 
         manifest.executionFunctions = new ManifestExecutionFunction[](3);
-        manifest.executionFunctions[0] =
-            ManifestExecutionFunction({executionSelector: this.onERC721Received.selector, isPublic: true});
-        manifest.executionFunctions[1] =
-            ManifestExecutionFunction({executionSelector: this.onERC1155Received.selector, isPublic: true});
-        manifest.executionFunctions[2] =
-            ManifestExecutionFunction({executionSelector: this.onERC1155BatchReceived.selector, isPublic: true});
+        manifest.executionFunctions[0] = ManifestExecutionFunction({
+            executionSelector: this.onERC721Received.selector,
+            isPublic: true,
+            allowDefaultValidation: false
```

Comment:
In the spirit of simplifying plugin dev work, `allowDefaultValidation` can be omitted in this case.

In reality, both of those two flags will default to false if not set. We should document this somewhere.